Bug #18668: Allow fast restarts

If domeins reboot to fast, Xend detects this and stopps the domain. This breaks SysPrep from UCS DVS.
--- a/tools/python/xen/xend/XendConstants.py	2010-06-15 14:01:30.000000000 +0200
+++ b/tools/python/xen/xend/XendConstants.py	2010-06-15 14:01:54.000000000 +0200
@@ -96,7 +96,7 @@
 ZOMBIE_PREFIX = 'Zombie-'
 
 """Minimum time between domain restarts in seconds."""
-MINIMUM_RESTART_TIME = 60
+MINIMUM_RESTART_TIME = 10
 
 RESTART_IN_PROGRESS = 'xend/restart_in_progress'
 DUMPCORE_IN_PROGRESS = 'xend/dumpcore_in_progress'
